blackbox/docs/general/user-defined-functions.rst

```diff
@@ -28,24 +28,24 @@ full syntax description.
 
 These functions can be created like so::
 
-    cr> CREATE FUNCTION subtract(integer, integer)
+    cr> CREATE FUNCTION my_subtract_function(integer, integer)
     ...  RETURNS integer
     ...  LANGUAGE JAVASCRIPT
-    ...  AS 'function subtract(a, b) { return a - b; }';
+    ...  AS 'function my_subtract_function(a, b) { return a - b; }';
     CREATE OK, 1 row affected  (... sec)
 
 .. hide:
 
-    cr> _wait_for_function('subtract(1::integer, 1::integer)')
+    cr> _wait_for_function('my_subtract_function(1::integer, 1::integer)')
 
 ::
 
-    cr> SELECT subtract(3, 1);
-    +----------------+
-    | subtract(3, 1) |
-    +----------------+
-    |              2 |
-    +----------------+
+    cr> SELECT my_subtract_function(3, 1);
```

Comment:
I'd also add the schema to make the built-in lookup explicit to follow best-practice. If users don't do that they're always risking that new built-ins that we add end up clashing with their functions and break their stuff.